safe/gui/tools/wizard_metadata.py

```diff
@@ -0,0 +1,108 @@
+# coding=utf-8
+"""
+InaSAFE Disaster risk assessment tool by AusAid **GUI InaSAFE Wizard Metadata.**
+
+ ** NOTE This file meant for temporary storing metadata to be later moved
+         to proper places.
+
+
+Contact : ole.moller.nielsen@gmail.com
+
+.. note:: This program is free software; you can redistribute it and/or modify
+     it under the terms of the GNU General Public License as published by
+     the Free Software Foundation; either version 2 of the License, or
+     (at your option) any later version.
+
+"""
+__author__ = 'qgis@borysjurgiel.pl'
+__revision__ = '$Format:%H$'
+__date__ = '12/04/2016'
+__copyright__ = ('Copyright 2012, Australia Indonesia Facility for '
+                 'Disaster Reduction')
+
+
+road_class_mapping = [{
```

Comment:
Is-it possible to translate these strings ? (at least in the UI, maybe not in the XML)